run.ps1

```diff
@@ -259,4 +292,4 @@ try {
 }
 finally {
     Remove-Module 'KoreBuild' -ErrorAction Ignore
-}
+}
```

Comment:
I think most of the team has their whitespace settings configured to _add_ missing trailing newlines. Can you do the same to avoid this churn going forward❔ No big reason to make a change in this PR.

Reply:
Updated my Sublime settings, this should stop happening now.